### ISSUE TITLE
assertion 0 failed at /Users/eyberg/go/src/github.com/nanovms/nanos/src/runtime/tuple.c:29  in get(); halt

### ISSUE THREAD

**Issue by quantumsheep** [nanovms/nanos#1537]
While doing tests around #1533, I came across the following error:
```
$ ops pkg load -l mysql --show-debug
booting /Users/qtmsheep/.ops/images/program ...
en1: assigned 10.0.2.15
2021-07-09T18:36:45.552932Z 0 [System] [MY-010116] [Server] /program (mysqld 8.0.25) starting as process 2
2021-07-09T18:36:45.608980Z 1 [System] [MY-013576] [InnoDB] InnoDB initialization has started.
ffff80000015fdb0:   ffffffff800379b9	(get + 0000000000000029/0000000000000043)
ffff80000015fdc0:   ffffffff8003b5ab	(filesystem_set_time + 000000000000003b/000000000000014b)
ffff80000015fe40:   ffffffff8003b74d	(filesystem_set_mtime + 000000000000008d/00000000000000aa)
ffff80000015fe80:   ffffffff8006d36c	(file_write + 000000000000032c/00000000000004b6)
ffff80000015ff10:   ffffffff80046a3b	(io_submit + 000000000000039b/000000000000049b)
ffff80000015ff80:   ffffffff8006b8f2	(syscall_debug + 00000000000001f2/0000000000000523)
ffff80000015ffd0:   ffffffff8006c020	(configure_syscalls + 0000000000000000/0000000000000224)
assertion 0 failed at /Users/eyberg/go/src/github.com/nanovms/nanos/src/runtime/tuple.c:29  in get(); halt
exit status 255
```

After some researches, it seems that `tagof(e)` returns `255` here in visibly some cases:
https://github.com/nanovms/nanos/blob/430a9fe3c97abed247ee50ccfecf3217eadf34b2/src/runtime/tuple.c#L19-L31

The call came from `filesystem_set_time`'s call to `get_number`:
https://github.com/nanovms/nanos/blob/430a9fe3c97abed247ee50ccfecf3217eadf34b2/src/tfs/tfs.c#L130

**Comment by sanderssj** [comment]
This should be fixed by #1535

**Comment by quantumsheep** [comment]
Confirmed.
